src/GaussianProcessEmulator.jl

```diff
@@ -56,36 +57,33 @@ $(DocStringExtensions.FIELDS)
 
 """
 struct GaussianProcess{FT<:AbstractFloat, GPM}
-    "training inputs/parameters; N_samples x input_dim"
-    inputs::Array{FT}
-    "training data/targets; N_samples x output_dim"
-    data::Array{FT}
+    "training inputs and outputs, data stored in columns"
+    input_output_pairs::PairedDataContainer{FT}
     "mean of input; 1 x input_dim"
     input_mean::Array{FT}
     "the Gaussian Process (GP) Regression model(s) that are fitted to the given input-data pairs"
+    sqrt_inv_input_cov::Union{Nothing, Array{FT, 2}}
+    "prediction type (`y` to predict the data, `f` to predict the latent function)"
```

Comment:
I think the descriptions here do not match (prediction type and sqrt_inv_input_cov are switched)

Reply:
Done